packages/react-ui-core/src/DropDown/DefaultAnchor.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../Button'
+
+export default class DefaultAnchor extends PureComponent {
```

Comment:
What do you think about renaming this to `AnchorButton` or `DropdownAnchorButton`? That would be a more descriptive name as to what this component is.  If you name is `DropdownAnchorButton` I would actually export this so the user can use if it they want.

Reply:
Fair enough. I'll make that change